jenkins_rock/rockcraft.yaml

```diff
@@ -45,7 +45,7 @@ parts:
       - default-jre-headless
       - git
     build-environment:
-      - JENKINS_VERSION: 2.427
+      - JENKINS_VERSION: 2.426.3
```

Comment:
Why's this?

Reply:
Trivy is complaining about 2.427 now

Reply:
2.426.3 has the fix for the problem Trivy complains about, and is also the latest LTS version

Reply:
Understood. Can you open a different PR for this change so that it can be easily tracked?